spec/features/course/unread_status_management_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'Announcements read/unread status management', type: :feature do
+  subject { page }
+
+  let!(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:first_user) { create(:user, role: :administrator) }
+    let(:second_user) { create(:user, role: :administrator) }
+    let(:course) { create(:course) }
+
+    describe 'sidebar' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.